hedera-node/src/main/java/com/hedera/services/store/tokens/unique/UniqueStore.java

```diff
@@ -33,6 +34,8 @@
 public interface UniqueStore extends TokenStore {
 
 	ResponseCodeEnum mint(final TokenID tId, String memo, RichInstant creationTime);
```

Comment:
do we need `mint` if we have `mintProvisionally`?

Reply:
I've refactored it so now It's mintOne - used to mint a single token and increment the serial number once